docs/markdown/Reference-manual.md

```diff
@@ -1492,6 +1492,10 @@ the following methods:
   strings, returns the first argument that passes the `has_argument`
   test above or an empty array if none pass.
 
+- `first_supported_link_argument(list_of_strings)` *(added 0.46.0)*, given a
+  list of strings, returns the first argument that passes the
+  `has_link_argument` test above or an empty array if none pass.
```

Comment:
'above' -> 'below' or just omit it?

Reply:
Good catch, it's actually a mistake copy/pasted from `first_supported_argument`. I fixed both now.